### DISCUSSION TITLE
Reglas para Regalos de PHP Mexico

### DISCUSSION THREAD

**Discussion by eruizdechavez** [phpmx/comunidad#9] [Ideas]
Para evitar futuras discusiones y malos entendidos, decidí escribir un borrador de propuesta para las reglas de los regalos de la comunidad. Espero recibir el feedback del Core Team, Consejo Directivo y resto de la comunidad 🙂 .

---

# Regalos de PHP Mexico

## Objetivos

El dar regalos en los diferentes eventos de la comunidad tiene dos objetivos:

1. Motivar a las personas que actualmente forman parte de la comunidad a estar activas y participar en la misma.
2. Captar nuevas personas para hacer que la comunidad siga creciendo y obtenga mas diversidad.

## Fuentes

Los regalos que se dan en los eventos de PHP Mexico pueden venir de diferentes fuentes. En primer lugar están los patrocinios, los cuales vienen de empresas o personas que han hecho alguna aportación económica o en especie a la comunidad, con el fin de apoyarla y al mismo tiempo obtener un espacio promocional. Les siguen los regalos de empresas y personas quienes solo quieren apoyar a la comunidad, sin interés en un espacio promocional. Finalmente están los regalos que se compran con los recursos económicos de la comunidad.

## Beneficios

Los niveles de Patrocinios, duración de estos, y los beneficios otorgados están especificados en  el documento de patrocinios, ubicado en ...; dichos beneficios tienen presidencia sobre lo escrito en este documento.

Las empresas o personas que apoyen a la comunidad con aportaciones diferentes a un patrocinio oficial (ver párrafo anterior) son enormemente apreciadas, y se les dará su respectivo reconocimiento de forma publica; dicho reconocimiento no debe confundirse con los espacios promocionales reservados para las empresas o personas con patrocinios oficiales.

Los regalos que provengan de los recursos económicos de la comunidad (Patreon, donaciones, etc.) no requieren ni otorgan un espacio promocional especifico.

## Reglas de participación

- Para participar en las diferentes dinámicas de la comunidad, bastara con hacer un retuit, repost, like, o similar de la publicación oficial en su medio preferido (Twitter, Facebook, Instagram, LinkedIn, etc.).
- Participar en varias redes sociales es algo que les animamos a hacer, y no descalificará a la persona que lo haga, pero solo se tomara en cuenta una participación por persona.
- Cada dinámica podrá especificar sus propias reglas de participación, las cuales tendrán precedencia a las aquí escritas. En caso que no se especifiquen reglas por dinámica, o dichas reglas estén incompletas, las reglas aquí escritas serán las que tengan precedencia.
- Para las dinámicas de eventos en vivo (presentaciones, Meetups, conferencias, etc.) la participación se cierra a las 24 horas de haber terminado dicho evento.
- Los regalos físicos sólo se enviarán en territorio Mexicano; los gastos del envío corren por cuenta de la comunidad.

**Comment by asfo** [comment]
Hay varios puntos que veo:

— Si se apoya a los que son activos en Slack, la comunidad se estancará en solo los activos.

Aquí me recuerda un chiste de @Gnuget cuando se empezaba con los PHPuntos diciendo que quien tenía la zapatilla se ganaba unas cazuelas...

Y sí, esto le daría impulso a los activos, PERO, haría que haya menos posibilidades de crecer la comunidad.

— Por otro lado, le das a los nuevos y los invitas a unirse, lo hacen, se llevan su premio o ven que pierden y se van...

Es aquí, donde yo creo que justamente debe partirse en dos los premios ...

Unos buscarían incentivar la actividad dentro del Slack y podría, incluso, pasar de las bromas a una realidad dando año con año un juego de "cazuelas" al top 1 de PHPuntos, por ejemplo... 

Y del otro lado, utilizar (sobre todo Facebook) las redes sociales para captar seguidores, los cuales, se pueden utilizar para impulsar todos los proyectos que se hagan internamente (caso Podcast, mentorías, web, etc...) y a su vez, incentivar la visualización de meetups...

Sobre todo este último, dado que si tenemos mayor alcance, podrá haber más interesados en dar un meetup, y a la vez atraemos mayor atención sobre estas dado que puede haber personas más conocidas en la comunidad, a nivel global, e incluso nuevos patrocinadores que permitan quizá el día de mañana incluso hacer eventos en línea (conferencias, talleres y demás).

Por eso a mi pensar creo que debemos formar una base sólida fuera de Slack sobre una red social que tenga mucho alcance y los giveaways pueden lograr esto (lo digo por experiencia propia), pero, sin tanta "tarea" que comúnmente se agrega "consigue las perlas de la vírgen, las esferas del dragón y dale la vuelta al mundo" cuando sólo debería ser "comparte en modo público y dale like a la página".

Es mi pequeña opinión :)

**Reply by smmd** [reply]
Me agrada tu punto de vista, el tema de los PHPuntos del slack podría usarse muy bien, pero creo debería "resetearse" cada cierto tiempo entonces, cada que se entregue un regalo por ejemplo.

**Reply by eruizdechavez** [reply]
Completamente de acuerdo @asfo! Precisamente por tu comentario en Slack fue que decidí incluir los dos objetivos, pero no puse ningún detalle para dejarlos abiertos a discusión. 

Así como lo veo, los regalos para atraer gente nueva deberían ser fáciles de conseguir (como dices, un solo post publico debería bastar) y los regalos más "pesados" para animar a personas nuevas a contribuir. También creo que alguien que acaba de ganarse un premio no debería poder participar dos veces seguidas, esto para darle la oportunidad a otras personas.

> Por eso a mi pensar creo que debemos formar una base sólida fuera de Slack sobre una red social que tenga mucho alcance 

Creo que hace falta aclarar este punto; por lo que entiendo, estaría dandole mas peso a Facebook (por poner un ejemplo) y esto podría potencialmente fracturar la comunidad. Por el contrario, si lo que significa es darle un mejor uso y aprovecharlo para dar mas difusión, entonces sí estoy de acuerdo.

**Reply by asfo** [reply]
@eruizdechavez Para lo segundo...

En resumen:
Hay que conseguir muchos likes + seguidores que orgánicamente, nos den mayor alcance en actividades

Y listo, así podemos sacar nuevas cosas y que de jalón tengan respaldo y no mueran en un "intento de sacar algo" y que por ello, futuros proyectos de la comunidad simplemente pierdan el interés.

**Comment by smmd** [comment]
Algo que tambien debemos tomar en cuenta es que los regalos físicos deben ser siempre enviados al interior de la Republica Mexicana y será un "disclaimer" que debemos tener VISIBLE todo el tiempo (no me vuelve a pasar que un elephpant se quede en la aduana y nos dejen en visto...)

**Reply by eruizdechavez** [reply]
No habíamos empezado a escribir algo así en las notas?

**Reply by eruizdechavez** [reply]
Ya lo encontré https://notes.erch.co/_C8AJp7JSqCk3BFuxfuE-Q

**Reply by eruizdechavez** [reply]
Acabo de actualizar el texto con las reglas que teníamos en el link de las notas 😉 .

**Comment by eruizdechavez** [comment]
@dmouse tenemos documentado en algún a detalle lugar (ademas de https://phpmexico.mx/job/plans) cómo funcionan los patrocinios?

**Comment by RZEROSTERN** [comment]
My 2 Bitcoin Cents:

Primeramente, el texto se ve comprensible y sin ningún problema.

Somo me gustaría agregar que con el uso de los hilos de Slack (como lo hemos hecho hasta hoy) podemos tener a verdaderos interesados en la comunidad y en el tema de la meetup en turno, además de ser un mecanismo bueno ahorita que llevamos pocos concursantes. Podríamos irlo afinando conforme crezca la comunidad.
